common/app/views/support/GetClasses.scala

```diff
@@ -58,7 +58,7 @@ object GetClasses {
       containerDefinition.showLatestUpdate,
       containerDefinition.index == 0 && containerDefinition.customHeader.isEmpty,
       containerDefinition.displayName.isDefined,
-      containerDefinition.displayName == Some("headlines"),
+      containerDefinition.displayName.contains("headlines"),
```

Comment:
Won't this fail on `scala 2.10`?
Locally I think this won't compile for me, as `Option` doesn't have a `contain` method before `2.10`.